### ISSUE TITLE
Screen textures (intro, title, menu, win cutscenes)

### ISSUE THREAD

**Issue by greeniporcini** [SuperNinjaFat/HDmod#189]
- [x]  Title Screen
- [ ] #427
- [x]  Desert Palm Trees and Desert Background (Day and Night)
✍️ Night version with Moon for special hell ending should swap to bg_duat moon texture (or change theme to duat idk) to not have the ripple effect from base_surface cosmos 
- [x] Hot Air Balloons
- [x] Giant idol
- [ ] #257
- [x] volcano and player escaping volcano sprites

**Comment by SuperNinjaFat** [comment]
What if we added in the bats from HD's title screen? 👀

**Comment by greeniporcini** [comment]
if i had the willpower once everything else is done. it's a massive maybe

**Comment by SuperNinjaFat** [comment]
I forget, did @Omeletttte ever mention wanting to sprite the camel?
Whoever does it, I'd say the amount of frames HD's camel has is lacking, so I'd be down with using the same frame count and poses as the Lamassu

**Comment by SuperNinjaFat** [comment]
> What if we added in the bats from HD's title screen? 👀

S2 actually already has menu bat sprites, nevermind!

**Comment by SuperNinjaFat** [comment]
Closed this issue since most items were completed and the Menu textures are postponed for a future update.
